index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4dc3d7e55b.js" crossorigin="anonymous"></script>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kanit:wght@300&display=swap" rel="stylesheet">
+    <link href="about.html" rel="import" /> 
+    
+    <title>Josué Benavides</title>
+</head>
+
+<body>
+    <div class="intro-container">
+        <header>
+            <div class="close">
+                <span class="cross"><i class="fa-solid fa-xmark"></i></span>
+                <ul class="close-list">
+                    <li class="close-item"><a href="#intro">Home</a></li>
+                    <li class="close-item"><a href="#program">Program</a></li>
+                    <li class="close-item"><a href="#speakers">Speakers</a></li>
+                    <li class="close-item"><a href="#sponsor">Sponsors</a></li>
+                    <li class="close-item"><a href="./about.html">About</a></li>
+                </ul>
+            </div>
+            <div class="nav-bar-mobile">
+                <div class="black-nav">
+                    <ul class="ul-black">
+                        <li class="github"><a href="" class="a"><i class="fa-brands fa-github"></i></a></li>
+                        <li class="facebook"><a href="" class="a"><i class="fa-brands fa-facebook"></i></a></li>
+                        <li class="anything"><a href="" class="a">Spanish</a></li>
+                    </ul>
+                </div>
+                <div class="mobile-logo">
+                    <button type="button" class="hamburger"><i class="fa-solid fa-bars"></i></button>
+                    <img src="./images/josue-logo.png" alt="josue-design" class="logo-img">
+                </div>
+                
+                <ul class="navigation">
+                    <li class="logo"><img src="./images/josue-logo.png" alt="josue-design" class="logo-img"></li>
+                    <li class="menu"><a href="#intro">Home</a></li>
+                    <li class="menu"><a href="#program">Program</a></li>
+                    <li class="menu"><a href="#speakers">Speakers</a></li>
+                    <li class="menu"><a href="#sponsor">Sponsors</a></li>
+                    <li class="about"><a href="./about.html">About</a></li>
+                </ul>
+            </div>
+        </header>
```

Comment:
- [ ] Kİndly use `nav` tag for navbars to use [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) tags.
- [ ] In the original design black navbar has more space at the top and bottom. Kİndly fix it.
- [ ] Second bar links should be about, program, join, sponsors, news, and cc campaign(with red border). Kindly change it.
- [ ] In your design second bar links close to the right side. Please follow the design layout(composition and space between elements.). Kindly change it.
![image](https://user-images.githubusercontent.com/38635158/176967917-adc992ea-7f5c-43d8-97bf-2eb979445362.png)
- [ ] In the mobile view it navbar has no logo. Kindly fix it.
- [ ] These sections are also on the about.html page so kindly change them on this page.
___
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/176967956-cccfabf2-b549-4166-acad-36ea0797709d.png)
___
### YOUR DESIGN MOBİLE VİEW

![image](https://user-images.githubusercontent.com/38635158/176968486-23c22891-36fa-4160-a4f7-1cc34e04099f.png)
___
### ORIGINAL MOBİLE VİEW
![image](https://user-images.githubusercontent.com/38635158/176968555-6656d0cb-6012-486f-94c5-f6404d476e6e.png)